testproject/tests/test_second_step_authentication.py

```diff
@@ -60,16 +94,15 @@ def test_wrong_second_step_verification_with_wrong_code(active_user_with_email_o
     client = APIClient()
     first_step = login(active_user_with_email_otp)
     response = client.post(
-        path="/auth/login/code/",
+        path=PATH_AUTH_JWT_LOGIN_CODE,
         data={
             "ephemeral_token": first_step.data.get("ephemeral_token"),
             "code": "test",
         },
         format="json",
     )
-    code_error = "invalid_code"
-    assert response.status_code == 400
-    assert response.data.get("non_field_errors")[0].code == code_error
+    assert response.status_code == 401
```

Comment:
in other tests we are using `rest_framework.status` const so we can do in this test too